src/EnergyPlus/SQLiteProcedures.cc

```diff
@@ -1827,9 +1822,8 @@ void SQLite::createSQLiteDaylightMapTitle(int const mapNum,
         sqliteBindText(m_daylightMapTitleInsertStmt, 2, mapName);
         sqliteBindText(m_daylightMapTitleInsertStmt, 3, environmentName);
         sqliteBindForeignKey(m_daylightMapTitleInsertStmt, 4, zone);
-        sqliteBindText(m_daylightMapTitleInsertStmt, 5, refPt1);
-        sqliteBindText(m_daylightMapTitleInsertStmt, 6, refPt2);
-        sqliteBindDouble(m_daylightMapTitleInsertStmt, 7, zCoord);
+        sqliteBindText(m_daylightMapTitleInsertStmt, 5, refPts);
+        sqliteBindDouble(m_daylightMapTitleInsertStmt, 6, zCoord);
```

Comment:
Realizing now that this changes the number of columns in the SQLite output. @jmarrec Is this ok?
It also puts 1:n daylighting reference point coordinates (which are just informative here, they don't really apply to the map values) into a single string field. e.g.
 "RefPt1=(2.50:2.00:0.80),RefPt2=(2.50:18.00:0.80),RefPt3=(2.50:18.00:0.50)" where before only the first two were written and they were in separate fields.

Reply:
It'd be great to have the OutputRulesChanges filled out so I could see what's going on a bit better. But you shouldn't be too concerned about changing stuff in the SQL, OpenStudio can adapt to the new format (provided we're aware of the need to do it)

When merging this, it'd be great to open an Enhancement Request on https://github.com/NREL/OpenStudio/issues to clearly document the need to adjust OpenStudio once it targets the next version of EnergyPlus (probably 9.6.0-IOFreeze) though, as we could easily miss it later down the road.